src/main/java/entities/User.java

```diff
@@ -0,0 +1,170 @@
+package java.entities;
+import java.util.ArrayList;
+
+public class User {
+    // ArrayList of User's favourited posts
+    private final ArrayList<Post> favourites;
```

Comment:
Change data type to List to keep it more general

Reply:
changed